testing/options/src/main/proto/com/toasttab/protokt/testing/options/custom_file_descriptor_object_name.proto

```diff
@@ -15,10 +15,12 @@
 
 syntax = "proto3";
 
-package com.toasttab.protokt.testing.options;
+package toasttab.protokt.testing.options;
```

Comment:
All of these files should be relocated to a directory not containing `com`, so in this case `src/main/proto/toasttab/protokt/...`.

Reply:
discussed offline and i'm gonna switch back to com to make diff more readable, and then follow up with moving files/directories around PR